css/selectors/matches.json

```diff
@@ -8,60 +8,125 @@
           "support": {
             "chrome": [
               {
-                "version_added": "66"
+                "version_added": "12",
+                "alternative_name": ":-webkit-any",
+                "notes": [
+                  "Doesn't support combinators."
+                ]
               },
               {
-                "version_added": "12",
-                "alternative_name": ":-webkit-any"
+                "version_added": "66",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform Features",
```

Comment:
With due respect to @ExE-Boss Google would prefer you use the link instead of the text for flags. The link is less likely to be changed.

Reply:
I'd like to see this filed as an issue and mentioned in the docs. We can then fix throughout and maybe even test this. @jpmedley @ExE-Boss

Reply:
https://github.com/mdn/browser-compat-data/issues/2992